autoload/lsp/utils/diff.vim

```diff
@@ -0,0 +1,115 @@
+" This is copied from https://github.com/natebosch/vim-lsc/blob/master/autoload/lsc/diff.vim
+"
+" Computes a simplistic diff between [old] and [new].
+"
+" Returns a dict with keys `range`, `rangeLength`, and `text` matching the LSP
+" definition of `TextDocumentContentChangeEvent`.
+"
+" Finds a single change between the common prefix, and common postfix.
+function! lsp#utils#diff#compute(old, new) abort
```

Comment:
Also update README.md that we support incremental changes.

Reply:
just a note in README.md is ok.

I need to spend time on making changelog autogenerated by something like this. https://github.com/git-chglog/git-chglog